features/s3Encryption/encryption.feature

```diff
@@ -0,0 +1,42 @@
+@s3Encryption @integ
+Feature: S3 Client Side Encryption
+
+  Scenario: Upload PHP's GCM encrypted fixtures
+    When I get all fixtures for "aes_gcm" from "aws-s3-shared-tests"
```

Comment:
Is this the name of a particular bucket? Including a resource identifier here seems fragile.

Reply:
It is a bucket name and it is fragile. This is how [the AWS SDK for Go handles it](https://github.com/aws/aws-sdk-go/blob/c47b5f826383ef61914397ce2ca736cd92365eee/awstesting/integration/customizations/s3/s3crypto/s3_crypto.feature#L6) and we do something similar for the [other full integration tests as well](https://github.com/aws/aws-sdk-php/blob/master/tests/Integ/S3Context.php#L45). These tests need to talk to other SDKs and don't get a `getResourcePrefix` added. I'm open to suggestions on this.